pulser-core/rtd_requirements.txt

```diff
@@ -0,0 +1 @@
+pulser-core/.
```

Comment:
What is this for ?

Reply:
This is needed for the Read The Docs builder to install `pulser-core`